docs/projects/t1-2022/teams.md

```diff
@@ -113,23 +113,22 @@ Product Lead: Yash Kondlekar
 
 ### Extensions
 
-- Teams Channel: TBD
+- Teams Channel: [Language Extensions](https://teams.microsoft.com/_?tenantId=d02378ec-1688-46d5-8540-1c28b5f470f6#/school/conversations/Language%20Extensions?threadId=19:b7feb53773244946936db2ee9df315f4@thread.tacv2&ctx=channel)
+- Trello Board: [Language Extensions](https://trello.com/b/xIVeBYwU/language-extensions)
 - Delivery Lead: Allan Farrell
 
 #### Python Compatibility
 
-- Trello Board: [Python Compatibility](https://trello.com/b/cZXHjIv0/python-compatability)
-- Team Member: Norbert Pop, Jack Sievers, Ben Thomas, Liu Sicheng
+- Team Member: Norbert Pop, Jack Sievers, Ben Thomas
 
 #### Language Extensions
 
-- Trello Board: [Language Extensions](https://trello.com/b/xIVeBYwU/language-extensions)
```

Comment:
The two boards have been consolidated into one. Could you confirm this @allanfarrell?